core/config/docs/docs_test.go

```diff
@@ -97,8 +97,10 @@ func TestDoc(t *testing.T) {
 		docDefaults.Transactions.AutoPurge.Threshold = nil
 		docDefaults.Transactions.AutoPurge.MinAttempts = nil
 
-		// GasEstimator.DAOracle.OracleAddress is only set if DA oracle config is used
-		docDefaults.GasEstimator.DAOracle.OracleAddress = nil
+		// Fallback oracle address must be an EIP55-compatible address
+		zeroAddress, err := types.NewEIP55Address("0x0000000000000000000000000000000000000000")
+		require.NoError(t, err)
+		docDefaults.GasEstimator.DAOracle.OracleAddress = &zeroAddress
```

Comment:
@jmank88 is this the correct way to manage the type differences between the docs default and fallback?

Reply:
IIRC these are exclusively exceptional cases. What happens if you remove this?